### ISSUE TITLE
Search large indexes "online"

### ISSUE THREAD

**Issue by colinmollenhour** [jreinke/magento-admin-launcher#1]
The local javascript model makes sense for small indexes like config and menus, but it would be really cool if large indexes like catalog, customers and orders could also be used without sacrificing performance.

Proposal: some indexes could be made "online" and online indexes would only be invoked if the indexer group prefix was used. E.g. "product: Widget" would invoke an online search for products, but "Widget" would only search the offline indexes (config, menus, etc.). So in a sense it would be like the global search, only much faster since it would be more targeted and with a much nicer UI.

**Comment by jreinke** [comment]
Yep I agree and I'll think about it.

At the moment launcher works well with thousands of entry. I tested with 2k products without any problem and I think this is enough for most of Magento stores. At least I hope so :)

**Comment by colinmollenhour** [comment]
Yes, 2k products seems ok, although with many tabs open memory usage might become an issue. However, if customers and orders were to be supported these can easily get to be in hundreds of thousands or millions. Also, when searching products it would be nice to utilize the power of full-text search since it is not always easy to remember the order of words in product names.
